array.js

```diff
@@ -0,0 +1,251 @@
+module.exports = {
+    array:{

```

Comment:
remove this extra nesting. Its unnecessary. Each array can be its own property u dont need the array